enterprise/server/remote_execution/execution_server/execution_server.go

```diff
@@ -998,6 +998,29 @@ func (s *ExecutionServer) MarkExecutionFailed(ctx context.Context, taskID string
 		log.CtxWarningf(ctx, "MarkExecutionFailed: error updating execution: %q: %s", taskID, err)
 		return err
 	}
+	err = func() error {
```

Comment:
nit: probably a little more conventional to extract a named top-level func like `recordFailedExecutionToClickHouse` or something like that